src/YQApplication.cc

```diff
@@ -773,9 +773,17 @@ void YQApplication::setApplicationIcon ( const string& icon )
 {
   QString qtIcon = fromUTF8( icon );
   YApplication::setApplicationIcon ( icon );
-  QPixmap pixmap (qtIcon);
-  if ( !pixmap.isNull() )
-    qApp->setWindowIcon ( QIcon ( pixmap ) );
+  QString icon_name = QFileInfo(qtIcon).baseName();
```

Comment:
No `#include <QFileInfo>`  ?

Reply:
Checked out headers needed, and it turns out that neither were needed to build.

Reply:
So it's probably indirectly included by some other Qt header. Let's just hope they don't rearrange those indirect includes anytime soon. ;-)